cmd/main.go

```diff
@@ -136,6 +142,7 @@ func main() {
 	flag.StringVar(&webhookCertDir, "webhook-cert-dir", "/tmp/k8s-webhook-server/serving-certs/",
 		"Webhook cert dir, only used when webhook-port is specified.")
 	flag.StringVar(&pprofBindAddress, "pprof-bind-address", "", "The TCP address that the controller should bind to for serving pprof, \"0\" or empty value disables pprof")
+	flag.BoolVar(&enableIPAM, "enable-ipam", true, "Enable cluster IPAM feature")
```

Comment:
i cannot find the corresponding values knob within the `kcm` chart to enable or disable the feature. should it be turned on by default though?